CONTRIBUTING.md

```diff
@@ -24,13 +23,13 @@ contributing, please read the
 ## Not sure where to start?
 
 - If you aren't just making a documentation change, you'll probably want to learn a bit about a few topics.
- - [ASTs](https://en.wikipedia.org/wiki/Abstract_syntax_tree) (Abstract Syntax Tree): The Babel AST [spec](https://github.com/babel/babel/blob/master/packages/babel-parser/ast/spec.md) is a bit different from [ESTree](https://github.com/estree/estree). The differences are listed [here](https://babeljs.io/docs/en/next/babel-parser.html#output).
- - Check out [`/doc`](https://github.com/babel/babel/tree/master/doc) for information about Babel's internals
- - Check out [the Babel Plugin Handbook](https://github.com/thejameskyle/babel-handbook/blob/master/translations/en/plugin-handbook.md#babel-plugin-handbook) - core plugins are written the same way as any other plugin!
- - Check out [AST Explorer](http://astexplorer.net/#/scUfOmVOG5) to learn more about ASTs or make your own plugin in the browser
+- [ASTs](https://en.wikipedia.org/wiki/Abstract_syntax_tree) (Abstract Syntax Tree): The Babel AST [spec](https://github.com/babel/babel/blob/master/packages/babel-parser/ast/spec.md) is a bit different from [ESTree](https://github.com/estree/estree). The differences are listed [here](https://babeljs.io/docs/en/next/babel-parser.html#output).
```

Comment:
@nicolo-ribaudo These changes were automatically made by the Markdown Formatter in my VSCode. Can you please guide me how to undo them?

Reply:
@nicolo-ribaudo Tried making changes and committed it but the formatter wrongly indented the starting HTML code. Which formatter should I use? Please help.

Reply:
You can temporarily disable the formatter extension.

Reply:
Okay, please review the changes made. Have made a new commit. Thanks.

Reply:
Oh, thanks! I didn't realize that there was only one space instead of 2 :sweat_smile: 
Your formatter was actually right and was showing that the old file has an indentation bug.